frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -328,7 +330,7 @@ export function DashboardItem({
                                     overlay={
                                         <Menu data-attr={'dashboard-item-' + index + '-dropdown-menu'}>
                                             <Menu.Item data-attr={'dashboard-item-' + index + '-dropdown-view'}>
-                                                <Link to={link}>
+                                                <Link to={link} onClick={() => setInsight(item)}>
```

Comment:
Curious why we're doing this, seems like we're still fetching the latest version from the API, and we might be better off using the latest version